inference/statnode.py

```diff
@@ -63,16 +64,15 @@ def extract_name(self):
                     name = entity.text
 
         return name
-    
+
     def extract_stat(self):
          doc = self.nlp(self.query)
          stat = ""
          stat_final = ""
-         max_sim = 0
-         sim_threshold = 0.7

```

Comment:
Let's also leverage a similarity threshold in case the query was misclassified and return `None` for the extracted stat if the threshold is not met.